ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -77,13 +77,22 @@ wait "$!"
 sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
 
 echo "Will include manifests:"
-find "${SHARED_DIR}" -name "manifest_*.yml"
+find "${SHARED_DIR}" \( -name "manifest_*.yml" -o -name "manifest_*.yaml" \)
```

Comment:
I personally prefer `.yaml` to `.yml`, but [`.yml` is part of our existing API][1].  If we want to pivot the CI-step consumer, I'd rather just move everything in the step registry `.yaml`, instead of supporting both.  If folks feel like that's too pendantic, feel free to override me, but also update the step docs to keep up.

[1]: https://github.com/openshift/release/blob/31fb393268d2e21f5fb68392e24f352d5bbea6ae/ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml#L15

Reply:
there is a  requirement in API for doing sts testing on AWS that one of the file in manifests must be `manifests/cluster-authentication-02-config.yaml` and therefore i need this to accept yml and yaml . I can update the doc test to include yaml.

Reply:
i don't really care if existing one are yml, no big deal supporting both.

Reply:
done added `yml|yaml` to doc text

Reply:
also updated to include tls_ slurping.